src/ui/database/add/AddDatabasePageContainer.js

```diff
@@ -1,12 +1,16 @@
 import { connect } from 'react-redux';
 import AddDatabasePage from 'ui/database/add/AddDatabasePage';
 import { fetchDatabaseInitBackup } from 'state/database/actions';
-
+import withPermissions from 'ui/auth/withPermissions';
+import { ROLE_SUPERUSER } from 'state/permissions/const';
 
 export const mapDispatchToProps = dispatch => ({
   onWillMount: () => dispatch(fetchDatabaseInitBackup()),
 });
 
 
-const AddDatabasePageContainer = connect(null, mapDispatchToProps)(AddDatabasePage);
+const AddDatabasePageContainer = withPermissions(ROLE_SUPERUSER)(connect(
```

Comment:
can you move `withPermissions(ROLE_SUPERUSER)` at the next line? (on `export defaults...`)